decidim-elections/app/views/decidim/elections/trustee_mailer/notification.html.erb

```diff
@@ -0,0 +1,6 @@
+<h2><%= t("subject", scope: "decidim.elections.admin.mailers.trustee_mailer", resource_name: @participatory_space_title) %></h2>
```

Comment:
```suggestion
```

Let's remove this line, please. We usually don't add `h2` in the mails. On first look, I thought that this was a bug: 


![image](https://user-images.githubusercontent.com/717367/161526435-1d2cf2df-7b0e-42c7-99e1-3eb3f1a25e47.png)

Reply:
I don't know where I got this from 🤔 . Probably copied code from somewhere else where this is misused.